src/styles/index.scss

```diff
@@ -1,3 +1,76 @@
+$cell-size: 100px;
+$day-count: 31;
+$col-count: 7;
+$gap-size: 1px;
+$padding-size: 10px;
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar {
+  display: flex;
+  padding: $padding-size;
+  flex-wrap: wrap;
+  gap: $gap-size;
+  width: $col-count * $cell-size + 6 * $gap-size + 2 * $padding-size;
```

Comment:
The calculation for the width of the calendar is correct, but ensure that the arithmetic operations are properly interpreted during SCSS compilation. If there are issues, consider using parentheses to clarify the order of operations.